content/chapter/02.tex

```diff
@@ -15,7 +15,7 @@ \section{Unwucht}
 
 \begin{figure}[H]
     \centering
-    \includegraphics[width=5cm]{images/chapter/02/static_imbalance.png}
+    \includegraphics[width=8cm]{images/chapter/02/static_imbalance.png}
```

Comment:
Ich würde die Bilder jeweils unter den Text schreiben. Prinzipiell bin ich dafür die Bilder imm vor der Beschreibung zu haben aber in dem Fall siehts doof aus und hilft auch erst nach der Beschreibung...